code/modules/research/xenoarchaeology/finds/finds_special.dm

```diff
@@ -150,6 +156,38 @@
 		B.blood_DNA[M.dna.unique_enzymes] = M.dna.b_type
 		M.vessel.remove_reagent(/singleton/reagent/blood,rand(25,50))
 
+///Is called when the statue is destroyed. Deals sufficiently consequential damage to IPCs and non-IPCS. Dionae are the least affected.
+/obj/item/vampiric/proc/bloodcall_final(var/mob/living/carbon/human/hero)
+	if(!istype(hero))
+		return
+	if(!istype(hero.species, /datum/species/machine))
+		to_chat(hero, SPAN_HIGHDANGER("What have you done?! You feel an incredible pull as the statue breaks. The force crushes your hands and pulls your blood into an unseen void."))
+		hero.apply_damage(40, DAMAGE_BRUTE, BP_L_HAND)
+		hero.apply_damage(40, DAMAGE_BRUTE, BP_R_HAND)
+		hero.vessel.remove_reagent(/singleton/reagent/blood,rand(120,175))
+		return
+
+	hero.apply_damage(700, DAMAGE_BRUTE, BP_GROIN)
+	hero.apply_damage(700, DAMAGE_BRUTE, BP_GROIN) //The way IPC limbs seem to act I needed the double damage to ensure it was removed.
```

Comment:
I think there is a droplimb proc if you want to ensure that a limb is removed.

Reply:
Thanks, I was able to rebalance with that proc. It should all be ready now.